src/WordSearch.java

```diff
@@ -0,0 +1,33 @@
+import java.io.File;
+
+/**
+ * Getting the file path from the command prompt
+ * calling the filePath method for searching the file in system
+ */
+public class WordSearch {
+    public static void main(String[] args) {
+        if (args.length == 2) {
+            String filepath = args[0];
+            String keyWordToSearch = args[1];
+            WordSearch wordsearch = new WordSearch();
+            wordsearch.processFile(filepath, keyWordToSearch);
+        } else {
+            System.out.println("Please pass the Arguments");
+        }
+    }
+
+    /*
+    Assigning the file path to File and checking file is Present in the system or not
+    */
+    public void processFile(String filepath, String keyWordToSearch) {
```

Comment:
Reform the method like public static void processFile(String filepath, String keyWordToSearch)
once reform not need to call like 
WordSearch wordsearch = new WordSearch();
            wordsearch.processFile(filepath, keyWordToSearch);
just add method not need to initialize an object
processFile(filepath, keyWordToSearch);

Reply:
Updated as Suggested.